app/workers/moderator/banish_user_worker.rb

```diff
@@ -0,0 +1,16 @@
+module Moderator
+  class BanishUserWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(admin_id, abuser_id)
+      abuser = User.find(abuser_id)
+      admin = User.find(admin_id)
+      Moderator::BanishUser.call(admin: admin, user: abuser)
+    rescue StandardError => e
+      DatadogStatsClient.count("moderators.banishuser", 1, tags: ["action:failed", "user_id:#{abuser.id}"])
+      Honeybadger.notify(e)
```

Comment:
In the future we might want to think about emailing the admin that it failed given that moderators might not have access to Honeybadger to know when and why a job failed.

Reply:
I feel like the infrastructure in the app we currently have is around Slack pings, but I'm actually not sure how admins would ideally want to ingest notices like this. Email makes sense, but it also makes sense to only have one way we do this.

@michael-tharrington @pkfrank thoughts?

Reply:
If we think it can be solved by an admin just hitting the button again, then I think a message to `#support` on Slack would be ideal. (But, an email notification to yo@ would also work.) 

However, if it's going to need someone technical to push it through, then `#tech-administration` would be the best option.